aptos-move/aptos-release-builder/src/main.rs

```diff
@@ -48,6 +52,12 @@ pub enum Commands {
         #[clap(long)]
         mint_to_validator: bool,
     },
+    /// Print out current values of on chain configs.
+    PrintConfigs {
+        /// Url endpoint for the desired network. e.g: https://fullnode.mainnet.aptoslabs.com/v1.
+        #[clap(short, long)]
```

Comment:
it would be nice if we could also accept a short name alias like 'testnet' 'mainnet' 'devnet' as well as a fully formed url